src/models/types.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Slate-specific item types.
+ *
+ * @type {Object}
+ */
+
+const TYPES = {
+  IS_SLATE_STATE: '@@__SLATE_STATE__@@',
```

Comment:
Can you change these to remove the `IS_SLATE` prefix? Since we're only using them internally, we don't need the extra verbosity.